web/models/user.ex

```diff
@@ -97,8 +98,8 @@ defmodule CodeCorps.User do
 
   def github_associate_changeset(struct, params) do
```

Comment:
Why not `github_association` changeset, since it looks like this wants to be a noun and `associate` in this case is verbed?